### PR TITLE
added gravatar profile picture to navbar and dash

### PR DESCRIPTION
**Pull request by dengel29**

### CODE REVIEW

File: app/views/dashboard/index.html.erb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <div class="main-content-container">
  <h1>Welcome to your Dashboard</h1>
+ <%= image_tag gravatar_url("you@youremail.com", 64), alt: "" %>
```
Comment (forresty) [inline app/views/dashboard/index.html.erb]: why not use current user email?

**Review by forresty — CHANGES_REQUESTED** [review]

**Comment by dengel29** [comment]
Ah, good catch I'm being lazy. I'll change that @forresty

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -18,7 +18,7 @@
       <ul class="nav navbar-nav navbar-right">
         <% if current_user %>
           <li class="dropdown">
-            <%= image_tag "http://kitt.lewagon.com/placeholder/users/ssaunier", class: "avatar dropdown-toggle", id: "navbar-wagon-menu", "data-toggle" => "dropdown" %>
+            <%= image_tag gravatar_url("you@youremail.com", 64), alt: "", class: "avatar dropdown-toggle", id: "navbar-wagon-menu", "data-toggle" => "dropdown" %>
```
Comment (forresty) [inline app/views/shared/_navbar.html.erb]: and this??? @dengel29

**Review by forresty — CHANGES_REQUESTED** [review]

**Review by forresty — APPROVED** [review]
LGTM

**Pull request merged**